pkg/kubelet/config/config.go

```diff
@@ -108,6 +110,19 @@ func (c *PodConfig) Sync() {
 	c.pods.Sync()
 }
 
+// Restore restores pods from the checkpoint path, *once*
+func (c *PodConfig) Restore(path string) error {
+	var err error
+	if c.checkpointManager == nil {
```

Comment:
if `c.checkpointManager` is not nil, will it skip the pod loading?

Reply:
Yes, it can only load once, this is in the offshoot some tries to call this twice.